frontend/src/store/creator/reducer.js

```diff
@@ -0,0 +1,140 @@
+import {
+	ADD_STEP,
+	REMOVE_STEP,
+	SELECT_EDGE,
+	SET_EDGE_ASSIGNMENT,
+	SET_EDGE_TARGET_ANGLE,
+	SET_FILE_AUTHOR,
+	SET_FILE_DESCRIPTION,
+	SET_FILE_TITLE,
+	SET_STEP_DESCRIPTION,
+	SET_STEP_TITLE,
+} from "./actions"
+
```

Comment:
Regarding reducer tests, we could probably copy-paste the tests generated from redux devtool.